CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-# Unreleased (Unreleased)
+# 0.7.0 alpha.1
+
+Currently the changelog for v0.7.0 is incomplete, this tracked in issue #1217.
```

Comment:
Suggestion (also currently missing "is"): `.. is incomplete. This is tracked in issue #1217.`

Reply:
Do you mind keeping it as is, as I already tagged the commit. Otherwise I have to force push the tag, which is something I rather not do.

Reply:
Ah yea no problem